core/src/program/blocks/mod.rs

```diff
@@ -37,6 +39,11 @@ impl CodeBlock {
         Self::Span(Span::new(operations))
     }
 
+    /// Returns a new Span block instantiated with the provided Span block parameters.
+    pub fn new_codeblock_from_span(span: Span) -> Self {
+        Self::Span(Span::from_span(span))
+    }
```

Comment:
If we implement the change from the previous comment, we can probably get rid of this.

Reply:
I've removed this code block in the latest PR